package.json

```diff
@@ -16,14 +16,17 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
+    "pretest": "node pretest.js",
     "test": "mocha --timeout 20000 --require test/init.js",
-    "posttest": "npm run lint"
+    "posttest": "node posttest.js && npm run lint"
```

Comment:
Mocha runs the entire test suite in the same process. It seems like there should be a way of doing a global before/after. Maybe it doesn't expose that functionality, though :-(